website/content/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways.mdx

```diff
@@ -42,6 +40,13 @@ gateways](/docs/connect/gateways/mesh-gateway/service-to-service-traffic-datacen
 
 [![WAN federation with mesh gateways](/img/wan-federation-connectivity-mesh-gateways.png)](/img/wan-federation-connectivity-mesh-gateways.png)
 
+## Requirements
+
+- Consul 1.8.0+
+- Mesh federation between Consul datacenters: all datacenters must be connected to all other datacenters
```

Comment:
```suggestion
- Mesh federation between Consul datacenters: all datacenters must be connected to all other datacenters
```
I'm a little confused by this bullet--isn't this topic about how to enact this requirement?

Reply:
I think I know what the confusion is. It's something I was worried about when writing this sentence, but didn't have an alternative ready at hand.

"Mesh" is being used for multiple things here, and that makes it confusing. I should probably describe this differently.

"Mesh federation" refers to a topology in which every datacenter is connected to every other datacenter. (I'm not sure this is a commonly understood term, but we do use it elsewhere in our docs.) You can accomplish "mesh federation" with or without "mesh gateways"... Which is confusing.

I think I need to remove the term "Mesh federation" from this. Maybe I can just simplify to:
```suggestion
- All Consul datacenters must be connected to all other datacenters
```

Thoughts?

Reply:
I think I understand now--the overloaded term "mesh" is probably a big reason why users have trouble understanding our docs. The change makes this a little clearer.

Reply:
> "Mesh federation" refers to a topology in which every datacenter is connected to every other datacenter. (I'm not sure this is a commonly understood term, but we do use it elsewhere in our docs.)

I'm unable to find any references to "mesh federation" on consul.io or Learn. Would you mind sharing where that terminology is used?

There are sections of the documentation, such as the [Advanced Federation](https://www.consul.io/docs/enterprise/federation) page, that state that Consul's traditional federation model requires that… "every datacenter be in a fully connected mesh." This referring to a [fully connected network](https://en.wikipedia.org/wiki/Network_topology#Fully_connected_network) which is a type of mesh network topology.

> the overloaded term "mesh" is probably a big reason why users have trouble understanding our docs.

I wouldn't consider the term to be overloaded, since its technically correct. I think that in order to reduce confusion between mesh networking topologies and service mesh, it might be better if our docs replace "fully connected mesh" with "fully connected network."

Reply:
I think in my memory "fully connected mesh" morphed into "mesh federation" for some reason. You're right.

I agree that "fully connected network" would be less confusing than "fully connected mesh" in this context in particular, and probably in other contexts in our docs as well.

Reply:
Changed to:

```
- All Consul datacenters must be connected to all other datacenters
   - Other topologies, such as "hub-and-spoke" enabled by Consul Enterprise network areas,
     are not supported
```